kmod/src/net.c

```diff
@@ -519,20 +519,20 @@ static int move_acked_responses(struct scoutfs_net_connection *conn,
 
 		msend->dead = 1;
 		list_move(&msend->head, &conn->send_queue);
-		ret = 1;
+		moved = true;
 	}
 
-	return ret;
+	return moved;
 }
 
 /* acks are processed inline in the recv worker */
 static void free_acked_responses(struct scoutfs_net_connection *conn, u64 seq)
 {
-	int moved;
+	bool moved;
 
 	spin_lock(&conn->lock);
 
-	moved = move_acked_responses(conn, &conn->send_queue, seq) +
+	moved = move_acked_responses(conn, &conn->send_queue, seq) |
```

Comment:
Just wondering if this is just to clean up some code, looks like before and after are logically the same.  I suppose if the ints could have been large they could've happened to overflow back to zero, but not here since the only possible values were 0 and 1.

Reply:
Yup, just code cleanup only.  It looked weird enough, but the change is safe enough, that I'm not worried about including it :).